agent/util-scripts/pbench-move-results

```diff
@@ -137,17 +148,20 @@ if [ $? -ne 0 ]; then
     debug_log "the following ssh command failed: \"ssh -q -i $pbench_bin/id_rsa $ssh_opts $results_repo exit\""
     exit 1
 fi
-ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "mkdir -p $results_full_path"
-if [ $? -ne 0 ]; then
-    error_log "ERROR: unable to create remote results path, $results_repo:$results_full_path"
-    exit 1
-fi
 
 let runs_copied=0
 let failures=0
 
-trap "rm -f $pbench_tmp/prefix.*" EXIT INT QUIT
+tmp=${pbench_tmp}/${script_name}.$$/
```

Comment:
There should not be a forward slash at the end because it is creating the wrong URL for the result.